cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -54,6 +54,10 @@ var (
 		Buckets: prometheus.ExponentialBuckets(.25, 2, 4), // 250ms -> 2s
 	}, []string{"success"}) // success=true|false
 
+	metricGetIndexOptions = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "get_index_options_total",
```

Comment:
nit: Didn't we have the convention to use "index_" for indexsever? I see we already have a mix of other prefixes. WDYT?

Reply:
we did, don't know what happened. But this is consistent with the metric this is paired with (the errors one) so rather be consistent. Renaming also means breaking the current alert, so is a bigger change to fix.